data/strings/strings.txt

```diff
@@ -5254,6 +5254,24 @@
     zh-Hans = 语音语言
     zh-Hant = 語音語言
 
+  [pref_tts_test_voice_title]
+    comment = Settings «Route» category: «Tts Test Voice» title
+    tags = android
+    en = Test voice
+    es = Probar voz
+    nl = Test stem
+    pt = Testar voz
+    pt-BR = Testar voz
+
+  [pref_tts_playing_test_voice]
+    comment = Settings «Route» category: «Playing Voice Test» pop-up message
+    tags = android
+    en = Playing voice test
```

Comment:
Let's make it more useful and more fun. What about creating several different strings, and playing a random string (ideally without repetition) on each keypress?

Even better, it can be a FAQ replacement! :)

You can propose your ideas too:

- Thank you for using our community-built maps!
- With your donations and support, we can create the best maps app in the World!
- Do you like our app? Please donate to support the development! Don't like it yet? Please let us know, and we will fix it!
- If you know a good software developer, you can ask him to implement a feature that you need.
- Do you know that you can long-tap any place on the map to select it?
- Do you know that you can select your position on the map?
- You can help to translate our app into your language.
- Our app is developed by a few enthusiasts and the community.
- You can easily fix and improve the map data.
- Our main goal is to build fast, privacy-focused, easy-to-use maps that you will love.

Reply:
Ok, this approach seems indeed funnier than mine... ;)

A few things that I'd like to confirm before I start to implement it:

1. I presume that these FAQ strings are brand new and they're not in some place in the code or in any website.

2. In order to have the "randomness" and "no repetition" behaviour when the pressing the "test button", here's a proposal of implementation:

    - In the first button click, the app will choose randomly one of the FAQ strings.
    - For the following clicks, the next FAQ string on the list will be played.

3. The pop-up message displayed when the button is pressed shall be always the same as now "Playing voice test"? Or shall we also print in this pop-up message the same string that we're playing?

Reply:
1. No. We have a long list of FAQ that needs to be published in the app and on our website, but there are no resources yet to do that. Any volunteering help is welcome, I can create a separate issue for it. While it would be great to reuse the FAQ, these TTS strings should be shorter, and may not contain lists, images, etc.
2. A true randomness is better. It may also make sense to save played strings to avoid their repetition until they all will be played. Take an array of string ids [tts_test_0, tts_test_1, tts_test_2, ...], then shuffle it, and play the array sequentially from the 0th element, while saving in settings the id of the played string.
3. Showing a text tooltip is a great idea for those impatient :)

It may also make sense to not complicate this task and merge it now with only one string, and create a separate issue/PR for the easter egg version to finish it later.

A similar approach can be later implemented in iOS.

Reply:
Commit updated with latest proposal: array is shuffled at start-up of the "Voice Instructions" settings' menu, and playbacks start at 0th element, but I don't see the need of storing anywhere the id of the played string.

I've only left one single voice test string. More voice test strings to be included in an additional PR.

Reply:
I've launched [PR 5974](https://github.com/organicmaps/organicmaps/pull/5974) for the addition of App Tip strings that could be used for the TTS voice testing.